internal/pkg/utils/testproject/snapshot.go

```diff
@@ -128,8 +128,7 @@ func (p *Project) snapshot(snapshot *fixtures.ProjectSnapshot, configs map[strin
 		Send()
 
 	// Wait for requests
-	err := pool.StartAndWait()
-	if err != nil {
+	if err := pool.StartAndWait(); err != nil {
```

Comment:
Tohle mi vyfailovalo až tady, lokální `make lint` si nestěžoval. 🤷‍♂️
![CleanShot 2022-01-31 at 14 27 57](https://user-images.githubusercontent.com/215660/151802532-a34ae598-5cbc-465c-84a6-35f3a28d840d.jpg)

Reply:
:man_shrugging: nevysla nova verzia nastrojov? (asi nie, myslim, ze je tam zamknuta verzia).
Tak to asi fixni, a ak by sa to opakovalo castejsie, tak sa na to pozrieme.

Reply:
resp. uz si to fixol.